README.md

```diff
@@ -93,17 +105,21 @@ Your Answer:
 
 What are the differences between calling and referencing a function? Please provide examples of each.
 
-```text
+calling a function initiates it runs the function when called. referencing only grabs the information of that function.
 
```

Comment:
:+1: just referring to a particular function, for example, when referencing a function during an event listener.

an example: 
```
function sayHello(){
  console.log("Hello, world!");
}
// Calling a Function:
sayHello();
// Referencing a Function:
var button = document.querySelector("button");
button.addEventListener("click", sayHello);
```